src/nyc_trees/apps/event/models.py

```diff
@@ -57,6 +57,16 @@ def training_summary(self):
     def has_space_available(self):
         return self.eventregistration_set.count() < self.max_attendees
 
+    @property
+    def starting_soon(self):
+        # TODO: Implement
+        return True
```

Comment:
I don't think it hurts to implement this now.  The time period of "soon" may not be set, but the logic of checking the `ends_at` date won't be changing